src/components/common.js

```diff
@@ -510,6 +510,11 @@ export const Switch = forwardRefWithName('Switch', ({ onChange, onLabel = 'True'
 export const HeroWrapper = ({ showMenu = true, bigSubhead = false, children }) => {
   const heavyWrapper = text => bigSubhead ? b({ style: { whiteSpace: 'nowrap' } }, [text]) : text
 
+  const endText = Utils.cond(
+    [isProjectSingular(), () => div(['and ', heavyWrapper('collaborate'), ' to advance research around single ventricle heart disease.'])],
```

Comment:
@kyuksel can you see about changing `heavyWrapper` so that it uses the `strong` tag instead of the `b` tag? With the HTML as-is, the screen reader experience is very poor. It stops reading before the first bold word, then you have to individually select each bit of text afterwards (and it doesn't state that it is bold). I haven't tried changing it to `strong`, but I'm hoping it will fix the issues.

https://help.siteimprove.com/support/solutions/articles/80000448460-accessibility-bold-vs-strong-and-italic-vs-emphasis

Reply:
Also, it feels odd to put the last part of the sentence in a different div. If the screen reader can read the whole thing seamlessly I guess it is OK, but semantically it feels wrong.

Reply:
Thanks for the catch! Addressed in [e1ce4a7](https://github.com/DataBiosphere/terra-ui/pull/3134/commits/e1ce4a7f3e3a57205a552839e3761913e205c83e).

Reply:
Agreed that `div` seems like the wrong element for wrapping a bit of text. I think `span` would be preferable. Does this need a DOM element at all?

Reply:
Also, does the `collaborate` styling need to be repeated in both branches of the condition? Could only the `to advance research around single ventricle heart disease` phrase be conditionally appended?

Reply:
> Agreed that div seems like the wrong element for wrapping a bit of text. I think span would be preferable. Does this need a DOM element at all?

Addressed in [fede1ef](https://github.com/DataBiosphere/terra-ui/pull/3134/commits/fede1efd03d410169d0c7c0d61d26ffaf56680fd)